serverlessworkflow/modules/ROOT/pages/getting-started/preparing-environment.adoc

```diff
@@ -29,22 +29,36 @@ Please note, that if the knative quickstart procedure is not used, you need to i
 .To startup the selected cluster without quickstart, use the following command:
 [tabs]
 ====
-Minikube::
+Minikube with Docker::
 +
 --
-.Configure and startup minikube
+.Configure and startup minikube with Docker
 [source,shell]
 ----
 # Set a driver and container runtime
-minikube config set driver docker/podman
-minikube config set container-runtime docker/podman
+minikube config set driver docker
```

Comment:
the guide was trying to have both paths, with docker and podman, but I deemed it unnecessary to split using tabs